yaml/pois.yaml

```diff
@@ -316,8 +316,10 @@ filters:
     output:
       kind: station
       state: {col: tags->state}
-  - filter:
-      railway: [halt, tram_stop, stop]
+  - filter: 
+      all:
+        railway: [halt, stop, tram_stop]
+        any: {historic: [-yes, -railway_stop], historic: false}
```

Comment:
Does this work as expected? I'm pretty sure that it wouldn't. Actually, I think a couple of things are going on here:

1. It would translate that first check into `historic IN ('-yes', '-railway')` rather than `historic NOT IN ('yes', 'railway')`
2. Because we're using the same key, `historic` twice, one would probably override the second. I'm guessing that just the `historic: false` rule is the one that would actually get applied.

We can spell it like:

```
  - filter:
      railway: [halt, stop, tram_stop]
      any:
        not:
          historic: ['yes', railway_stop]
        historic: false
```